### PR TITLE
Remove rules supported by eslint

### LINKED ISSUE #6: Add support for `jscs` 3.x.x

### LINKED ISSUE #8: Reduce preset to essential rules
The rules on this preset should be reduced to just those that are currently not supported by `eslint`.

### PR DESCRIPTION
**Pull request by ruiquelhas**
Resolves #8.
Resolves #6.

Refs: seegno/eslint-config-seegno#45

### CODE REVIEW

File: src/presets/seegno.json [hunk 1]
```diff
@@ -1,74 +1,23 @@
...
-  "disallowSpaceBeforePostfixUnaryOperators": true,
-  "disallowSpaceBeforeSemicolon": true,
-  "disallowSpacesInCallExpression": true,
-  "disallowSpacesInFunctionDeclaration": {
-    "beforeOpeningRoundBrace": true
-  },
-  "disallowSpacesInsideArrayBrackets": true,
-  "disallowSpacesInsideParentheses": true,
-  "disallowSpacesInsideParenthesizedExpression": true,
-  "disallowSpacesInsideTemplateStringPlaceholders": true,
```
Comment (rplopes) [inline src/presets/seegno.json:40]: Can't find this in the description of https://github.com/seegno/eslint-config-seegno/pull/45.

File: src/presets/seegno.json [hunk 2]
```diff
@@ -1,74 +1,23 @@
...
-  "disallowTrailingComma": true,
-  "disallowTrailingWhitespace": true,
-  "disallowYodaConditions": true,
-  "esnext": true,
```
Comment (rplopes) [inline src/presets/seegno.json:44]: Can't find this in the description of https://github.com/seegno/eslint-config-seegno/pull/45.

File: src/presets/seegno.json [hunk 3]
```diff
@@ -96,49 +44,5 @@
...
-    "switch",
-    "try",
-    "typeof",
-    "while",
-    "with"
-  ],
-  "requireSpaceAfterLineComment": true,
-  "requireSpaceBeforeBinaryOperators": true,
-  "requireSpaceBeforeBlockStatements": true,
-  "requireSpaceBeforeDestructuredValues": true,
```
Comment (rplopes) [inline src/presets/seegno.json:121]: Can't find this in the description of https://github.com/seegno/eslint-config-seegno/pull/45.

Comment (ruiquelhas) [inline src/presets/seegno.json:121]: It should be there now.

File: src/presets/seegno.json (hunk 2 above)
Comment (ruiquelhas) [inline src/presets/seegno.json:44]: Should be there now.

File: src/presets/seegno.json (hunk 1 above)
Comment (ruiquelhas) [inline src/presets/seegno.json:40]: Should be there now.

**Comment by rplopes** [comment]
LGTM

**Comment by rplopes** [comment]
As soon as https://github.com/seegno/eslint-config-seegno/pull/45 gets merged, this should be good to go too.

Either that, or updating the version number, to avoid projects being in a state where these rules no longer apply here but also don't apply yet in eslint.

**Comment by ruiquelhas** [comment]
@rplopes this preset is still not being used. This will be a replacement for [jscs-config-seegno](https://github.com/seegno/jscs-config-seegno). Not sure if it is worth a major release though.

**Comment by rplopes** [comment]
@ruiquelhas oh ok, my bad then. So it should be ok to merge now.

**Comment by rplopes** [comment]
After the latest changes, LGTM.

**Comment by ruiquelhas** [comment]
@franciscocardoso please give this a look, and if everything looks good, please merge!

File: src/presets/seegno.json [hunk 4]
```diff
@@ -96,49 +44,5 @@
     "while",
     "with"
   ],
-  "requireParenthesesAroundIIFE": true,
-  "requireSemicolons": true,
-  "requireShouldAssertionExecution": true,
-  "requireSpaceAfterBinaryOperators": true,
```
Comment (franciscocardoso) [inline src/presets/seegno.json:102]: Missing?

Comment (ruiquelhas) [inline src/presets/seegno.json:102]: Should be there now.

**Pull request merged**